Bottom: ed5f8b2d65a70d1b508c400e0b98dae46dcfcca3
Top:    904df4efcaa82ecdf00d70bf3dde1dafda6af2df
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 18:57:42 +0000

Refresh of improve-grid-snapping-heuristi

---

diff --git a/src/search.c b/src/search.c
index 5e86cc1..85c8fe4 100644
--- a/src/search.c
+++ b/src/search.c
@@ -1606,7 +1606,7 @@ SearchScreen (LocationType X, LocationType Y, int Type, void **Result1,
 }
 
 /* ---------------------------------------------------------------------------
- * searches the cursor position for the type 
+ * searches the cursor position for the type
  */
 int
 SearchScreenGridSlop (LocationType X, LocationType Y, int Type, void **Result1,
